src/components/DeckBoard.js

```diff
@@ -63,54 +83,189 @@ class DeckBoard extends React.Component {
       }
     }
 
-    this.setState({ deck: heroesChosenRandomized });
+    this.setState({ [deckName]: heroesChosenRandomized });
+  }
+
+  createIaDeck = () => {
+    const heroes = _.shuffle(this.props.heroes);
+    let IaDeckPower = 0;
+    const cardsAvalaibleForIA = this.state.cardsAvalaibleForIA;
+    for (let i = 0; i < this.props.heroes.length; i++) {
```

Comment:
rajouter la condition IaDeckPower <= this.props.maxPower dans la condition de terminaison du for, au lieu de faire un break